namespace cpp entity
namespace py ccentity.vocation
namespace as3 com.dreamfire.sanguo.data.definition.entity

enum VocationType {
  MIN = 0,

  AXE = 0,            // 近战
  SHOOT,              // 远程
  MAGIC,              // 法术

  MAX,
}

